### PR TITLE
Adds configurable opening delay + fade in animation. Adjusts the styles a bit.

### PR DESCRIPTION
**Pull request by juuso-vaadin**
The opening delay is controlled in theme via CSS variable. The same could be achieved programmatically.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -138,23 +138,21 @@ <h2 slot="header">Rich</h2>
             <style>
               [part='tooltip-title'],
               [part='tooltip-description'] {
-                text-align: left;
-                padding: 5px;
-                width: 100%;
+                padding: 0 8px;
```
Comment (lkraav) [inline demo/index.html:141]: Why wouldn't padding use Lumo sizing units here?

File: demo/index.html [hunk 2]
```diff
@@ -138,23 +138,21 @@ <h2 slot="header">Rich</h2>
...
+                font-style: italic;
+                margin-bottom: 0;
               }
 
             </style>
             <input type="text" id="rich-text-field" value="Text" />
             <vcf-tooltip for="rich-text-field" position="bottom" align="left" close-button>
-              <div part="tooltip-title">Edit</div>
-              <div part="tooltip-description">
+              <h3 part="tooltip-title">Edit</h3>
```
Comment (lkraav) [inline demo/index.html:152]: Specific heading level maybe over-opinionated?

How about `<label>` element instead?

File: demo/index.html (hunk 1 above)
Comment (jcgueriaud1) [reply demo/index.html]: It's changed. But it's only used in the demo, that's probably the main reason :)

File: demo/index.html (hunk 2 above)
Comment (jcgueriaud1) [reply demo/index.html:152]: It's used only in the demo, so it's opinionated.
<label> should only be used with a form element, it's not the case here.

**Pull request merged**